### DISCUSSION TITLE
Resizing fontSize on state changing for createTextColumn/textColumn

### DISCUSSION THREAD

**Discussion by flamenji** [nick-keller/react-datasheet-grid#220] [Q&A] [answered]
I am helped with this module a lot. Thank You very much.
But in any chance, is there any way to resize font with using useState for _createTextColumn_ and _textColumn_.

let's say I have Grid with
_selectColumn_ and _createTextColumn_

```
<DynamicDataSheetGrid
            style={{
              '--dsg-selection-border-color': 'tomato',
              '--dsg-background': '#fff1f0',
              fontSize: fontSize
            }}
            columns={columns}
            value={data}
/>
```

then I have a textBox Component which is 
`const [fontSize, setFontSize] = useState(14)`

perhaps can I modify `.dsg-input ` dynamically by using state?

But that state only affects _fontSize_ styling for _selectColumn_ and _rowHeader_ only

if any Sir you could give some leads, thank you

**Comment by nick-keller** [comment, marked as answer]
Hey @flamenji,
You could:
```jsx
<DynamicDataSheetGrid
  style={{
    // ...
    '--font-size': fontSize,
  }}
/>
```

And then in your CSS:
```css
.dsg-container {
  font-size: var(--font-size);
}

.dsg-input {
  font-size: var(--font-size);
}
```

**Comment by flamenji** [comment]
it works! thanks a bunch @nick-keller
